test/functional/gcp_cloud_storage_resources_exist.go

```diff
@@ -0,0 +1,151 @@
+package functional
```

Comment:
I don't mind leaving this test file in, but it's worth noting that the plan is to use NooBaa PV's for on-cluster s3 support. We won't be installing buckets into Google Cloud Storage for RHOAM so I'm not sure how we'll verify / update this.

Reply:
I added comment to file (just copied from your notes) + I commented GCP01 test